api/api.go

```diff
@@ -215,103 +215,113 @@ func NewServerRoutes(s Server) *mux.Router {
 	m.Use(refStringMiddleware)
 	m.Use(token.OAuthTokenMiddleware)
 
+	dsh := NewDatasetHandlers(s.Instance, cfg.API.ReadOnly)
+	proh := NewProfileHandlers(s.Instance, cfg.API.ReadOnly)
+	sqlh := NewSQLHandlers(s.Instance, cfg.API.ReadOnly)
+	tfh := NewTransformHandlers(s.Instance)
+	rch := NewRegistryClientHandlers(s.Instance, cfg.API.ReadOnly)
+	sh := NewSearchHandlers(s.Instance)
+	renderh := NewRenderHandlers(s.Instance)
+	remClientH := NewRemoteClientHandlers(s.Instance, cfg.API.ReadOnly)
+
 	var routeParams refRouteParams
 
+	// misc endpoints
 	m.Handle(lib.AEHome.String(), s.NoLogMiddleware(s.HomeHandler))
 	m.Handle(lib.AEHealth.String(), s.NoLogMiddleware(HealthCheckHandler))
 	m.Handle(lib.AEIPFS.String(), s.Middleware(s.HandleIPFSPath))
-
-	proh := NewProfileHandlers(s.Instance, cfg.API.ReadOnly)
-	m.Handle(lib.AEMe.String(), s.Middleware(proh.ProfileHandler))
-	m.Handle(lib.AEProfile.String(), s.Middleware(proh.ProfileHandler))
-	m.Handle(lib.AEProfilePhoto.String(), s.Middleware(proh.ProfilePhotoHandler))
-	m.Handle(lib.AEProfilePoster.String(), s.Middleware(proh.PosterHandler))
-
-	m.Handle(lib.AEPeers.String(), s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "peer.list"))).Methods(http.MethodPost)
-	m.Handle(lib.AEPeer.String(), s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "peer.info"))).Methods(http.MethodPost)
-	m.Handle(lib.AEConnect.String(), s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "peer.connect"))).Methods(http.MethodPost)
-	m.Handle(lib.AEDisconnect.String(), s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "peer.disconnect"))).Methods(http.MethodPost)
-	m.Handle(lib.AEConnections.String(), s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "peer.connections"))).Methods(http.MethodPost)
-	m.Handle(lib.AEConnectedQriProfiles.String(), s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "peer.connectedqriprofiles"))).Methods(http.MethodPost)
-
-	if cfg.Remote != nil && cfg.Remote.Enabled {
-		log.Info("running in `remote` mode")
-
-		remh := NewRemoteHandlers(s.Instance)
-		m.Handle(lib.AERemoteDSync.String(), s.Middleware(remh.DsyncHandler))
-		m.Handle(lib.AERemoteLogSync.String(), s.Middleware(remh.LogsyncHandler))
-		m.Handle(lib.AERemoteRefs.String(), s.Middleware(remh.RefsHandler))
+	if !cfg.API.DisableWebui {
+		m.Handle(lib.AEWebUI.String(), s.Middleware(WebuiHandler))
 	}
 
-	dsh := NewDatasetHandlers(s.Instance, cfg.API.ReadOnly)
+	// aggregate endpoints
 	m.Handle(lib.AEList.String(), s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "dataset.list"))).Methods(http.MethodPost, http.MethodGet)
 	m.Handle(lib.AEListRaw.String(), s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "dataset.listrawrefs"))).Methods(http.MethodPost, http.MethodGet)
 	m.Handle(lib.AEPeerList.String(), s.Middleware(dsh.PeerListHandler(lib.AEPeerList.String())))
```

Comment:
This should also merge into `AEList`

Reply:
Isn't this "/peer/list"? It's still there in https://github.com/qri-io/qri/pull/1697.

Reply:
So this particular endpoint is not the list of peers, but instead the list of datasets you have from a specific peer. Which I believe would become `/list` w/ the `filter=ramfox` in the params

Reply:
Yes what @ramfox said.
- `/peer/list` lists your peers
- `/list/{peer}` lists datasets of a given peer and should be moved into list with the new filtering approach